avocado/core/job.py

```diff
@@ -198,7 +198,7 @@ def __exit__(self, _exc_type, _exc_value, _traceback):
 
     def __start_job_logging(self):
         # Enable test logger
-        fmt = ('%(asctime)s %(module)-16.16s L%(lineno)-.4d %('
+        fmt = ('%(asctime)s %(name)-25.25s L%(lineno)-.4d %('
```

Comment:
We can have multiple modules in different packages using the same name, so having only the module name is not so helpful.

Also, multiple modules are using the 'avocado.test' namespace and we are using that information for figuring out what should be printed and where.

However, I agree with you that cutting this is not good. I was also bothered thinking that a huge name could be trimmed, and you just gave me an example.

IMO sounds strange to have a fixed length size inside the log file and I'm assuming that is to get a "nice" output if we would like to see only the "message". But this could be done using a cut by the "|" (pipe) instead of a cut by an exact column.

So a see two alternatives: 1) increase the length (which will be wasted on a small namespace); 2)  remove the namespace fixed length, showing the entire namespace. This will make the message to not be aligned as it is today, but is the way logs are and I don't see much as a problem.

Let me know what you think, but also keep in mind that I would like to provide soon a better way to users configure the formatters as well. (Post LTS)

Reply:
I am not sure of the options still, because long namespaces will then shift every log line further to the right (in a fix manner or not), occupying large screen real estate only printing identical initial information like `avocado.test` (assuming most of the logging happens from there which is mostly the case for all practical uses). Also considering plugin developers are down the supply chain, they would inevitably introduce new name spaces if they want to be able to debug and these name spaces will inevitably be subspaces of `avocado.test` or similar, leading to long names for everybody.

But allowing for full namespace length seems like the lesser evil and the only solution that could satisfy both points of view. I guess future formatting functionality could help even though there still isn't any optimal choice for the people that use it.

Reply:
Generic log filtering by `avocado.test` keywords also won't be very useful since we will end up with 95% if not all lines (rendering the filtering useless) but a good point for the `name` is that we could all then do log-postprocessing on top and there is at least a marginal improvement of filterability here.

Reply:
I will consider the comments here as acceptance of the proposal. Thanks.